modules/mono/glue/runtime_interop.cpp

```diff
@@ -304,6 +308,14 @@ void godotsharp_internal_new_csharp_script(Ref<CSharpScript> *r_dest) {
 	memnew_placement(r_dest, Ref<CSharpScript>(memnew(CSharpScript)));
 }
 
+void godotsharp_internal_editor_file_system_update_file(const String *p_script_path) {
+#if TOOLS_ENABLED
+	EditorFileSystem::get_singleton()->update_file(*p_script_path);
```

Comment:
It should be fixed now. I check if `EditorFileSystem::get_singleton()` returns `nullptr` and in that case the method does nothing, because when the `EditorFileSystem` becomes available it will update the files itself, so it should be fine.

Reply:
Resolved.